app/views/screens/LoginScreen.js

```diff
@@ -1,19 +1,19 @@
-import React, { useState } from "react";
+import React, { useState, useContext } from "react";
 import { Image, StyleSheet, Text, View } from "react-native";
 import { Button, TextInput } from "react-native-paper";
 
 import colors from "../styles/colors";
 import defaultStyles from "../styles/styles";
```

Comment:
How about we create one file under `styles` folder say `index.ts` and import every styles in that file.
So that we just need to import index.ts file in every file where we need these styles files. (This will help to reduce file size once our codebase grows)
~Like
```
// index.ts
import colors from "../styles/colors";
import defaultStyles from "../styles/styles";

// In other file 
import index from "../styles/styles.index.ts"
```

Reply:
I liked this idea. Thanks for suggesting! I will make these changes and update this PR.

Reply:
Updated PR